blueprints/dashboards/flow-control/quota-scheduler/dashboard.libsonnet

```diff
@@ -0,0 +1,187 @@
+local barGaugePanel = grafana.barGaugePanel;
+local statPanel = grafana.statPanel;
+local graphPanel = grafana.graphPanel;
+
+
+local newGraphPanel(title, datasource, query, axisLabel='', unit='') =
+  graphPanel.new(
+    title=title,
+    datasource=datasource,
+    labelY1=axisLabel,
+    formatY1=unit,
+  )
+  .addTarget(
+    prometheus.target(
+      expr=query,
+      intervalFactor=1,
+    )
+  );
+
+local newBarGaugePanel(graphTitle, datasource, graphQuery) =
+  local target =
+    prometheus.target(graphQuery) +
+    {
+      legendFormat: '{{ instance }} - {{ policy_name }}',
+      format: 'time_series',
+      instant: false,
+      range: true,
+    };
+
+  barGaugePanel.new(
+    title=graphTitle,
+    datasource=datasource,
+  ).addTarget(target) +
+  {
+    fieldConfig: {
+      defaults: {
+        color: {
+          mode: 'thresholds',
+        },
+        mappings: [],
+        thresholds: {
+          mode: 'absolute',
+          steps: [
+            { color: 'green', value: null },
+          ],
+        },
+      },
+      overrides: [],
+    },
+    options: {
+      displayMode: 'gradient',
+      minVizHeight: 10,
+      minVizWidth: 0,
+      orientation: 'horizontal',
+      reduceOptions: {
+        calcs: ['lastNotNull'],
+        fields: '',
+        values: false,
+      },
+      showUnfilled: true,
+    },
+  };
+
+local newStatPanel(graphTitle, datasource, graphQuery) =
+  local target =
+    prometheus.target(graphQuery) +
+    {
+      legendFormat: '{{ instance }} - {{ policy_name }}',
+      editorMode: 'code',
+      range: true,
+    };
+
+  statPanel.new(
+    title=graphTitle,
+    datasource=datasource,
+  ).addTarget(target) +
+  {
+    fieldConfig: {
+      defaults: {
+        color: {
+          mode: 'thresholds',
+        },
+        mappings: [],
+        thresholds: {
+          mode: 'absolute',
+          steps: [
+            { color: 'green', value: null },
+          ],
+        },
+      },
+      overrides: [],
+    },
+    options: {
+      colorMode: 'value',
+      graphMode: 'none',
+      justifyMode: 'center',
+      orientation: 'horizontal',
+      reduceOptions: {
+        calcs: ['lastNotNull'],
+        fields: '',
+        values: false,
+      },
+      textMode: 'auto',
+    },
+  };
+
+local dashboardWithPanels(dashboardParams, filters) =
+  {
+    local datasource = dashboardParams.datasource,
+    local dsName = datasource.name,
+
+    local WFQSchedulerFlows =
+      newBarGaugePanel('WFQ Scheduler Flows', dsName, 'avg(wfq_flows_total{%(filters)s})' % { filters: filters }),
+
+    local WFQSchedulerHeapRequests =
+      newBarGaugePanel('WFQ Scheduler Heap Requests', dsName, 'avg(wfq_requests_total{%(filters)s})' % { filters: filters }),
+
+    local TotalBucketLoadSchedFactor =
+      newStatPanel('Average Load Multiplier', dsName, 'avg(token_bucket_lm_ratio{%(filters)s})' % { filters: filters }),
+
+    local IncomingConcurrency =
```

Comment:
Some of these panels seem to be present in adaptive_load_scheduler also so we should use it instead of recreating it here